Runigram.java

```diff
@@ -42,7 +59,19 @@ public static Color[][] read(String fileName) {
 		// creates from the 3 colors a new Color object, and 
 		// makes pixel (i,j) refer to that object.
 		//// Replace the following statement with your code.
-		return null;
+		int r,g,b;

```

Comment:
There's no need to declare variables outside of it use. Read Java Style Guideline in the moodle.